src/main/java/net/mcreator/ui/workspace/ModElementCodeDropdown.java

```diff
@@ -61,26 +60,23 @@ class ModElementCodeDropdown extends JPopupMenu {
 
 			for (GeneratorTemplatesList fileList : modElementListFiles) {
 				if (fileList.templates().size() > 0) {
-					JMenu listMenu = new JMenu("<html>" + fileList.groupName());
+					JMenu listMenu = new JMenu(

```

Comment:
This is not ok as some of list conditions may fail and not generate for some of the list data thus making lower number of entries. Please revert the change

Reply:
You added a counter that is incremented each time next item from list data is picked, which would result in having a number equal to the size of that list data. If you instead intended this to show the amount of files generated, then you added the counter to a wrong lambda.

Reply:
`beforeNextListItem` is ran regardless of generation conditions applicable to templates being iterated.

Reply:
Hmm, I wanted to show number of list entries actually generated, but seems this is not possible then

Reply:
I will figure something out or remove this, I also found out I would need to externalize this string

Reply:
Then we'd need to use L10N for group names as well. Also, I pushed the change that I guess does what you wanted it to do.

Reply:
I actually just wanted to make height same and wanted to put something below to match height, but border can also do that so I will go with this solution